htop.c

```diff
@@ -270,22 +264,16 @@ static void setCommFilter(State* state, char** commFilter) {
 int main(int argc, char** argv) {
 
    char *lc_ctype = getenv("LC_CTYPE");
-   if (lc_ctype != NULL) {
+   if (lc_ctype != NULL)
       setlocale(LC_CTYPE, lc_ctype);
-   } else if ((lc_ctype = getenv("LC_ALL"))) {
+   else if ((lc_ctype = getenv("LC_ALL")))
       setlocale(LC_CTYPE, lc_ctype);
-   } else {
+   else
       setlocale(LC_CTYPE, "");
-   }
```

Comment:
`if` statements with an else clause should keep braces (cf. styleguide).

Reply:
Hmm - guess I've missed this part of the styleguide before.  I disagree with that being in the styleguide - its not consistent with plenty of existing code (which *is* pretty inconsistent though TBH) and IMO, in this simple case, it introduces syntactic noise that clutters the code and reduces readability.  Let's discuss that elsewhere though - I'll send a follow up PR to the styleguide for wider discussion.

Reply:
https://github.com/htop-dev/htop/pull/336